android-env.mk.inc

```diff
@@ -1,2 +1,3 @@
 export ANDROID_SDK_ROOT := /opt/android-sdk
 export ANDROID_NDK_HOME := /opt/android-sdk/ndk/21.2.6472646
+export GOMODCACHE_ROOT := /tmp/gomodcache/pkg/mod
```

Comment:
Why this? Is it required? If so, please add a comment why.

Reply:
Added a comment.